UITests/ReadingListTest.swift

```diff
@@ -40,7 +40,7 @@ class ReadingListTests: KIFTestCase, UITextFieldDelegate {
     func waitForEmptyReadingList() {
         let readable = GREYCondition(name: "Check readable list is empty", block: { _ in
             var error: NSError?
-            let matcher = grey_allOf([grey_accessibilityID("ReadingListEmptyPanel"),
+            let matcher = grey_allOf([grey_accessibilityLabel("Save pages to your Reading List by tapping the book plus icon in the Reader View controls."),
```

Comment:
These tests being white-box should be able to use Strings.SomeReaderModeString. But, this particular string is not exposed that way; in future (for me also to find out, because i have run into this) we should find out if localized strings should always be in the Strings.swift file (and then can use the variable here instead of literal).

Reply:
Agreed, I wonder whether we can raise a bug for issues like this.